### ISSUE TITLE
Add a setting for a list of folders to exclude

### ISSUE THREAD

**Issue by Ryex** [sytone/foundry-vtt-journal-sync#1]
This module causes problems with https://github.com/Forien/foundryvtt-forien-quest-log

the quest log assumes that the journal content stays valid JSON but on reimport, the content is wrapped with `<p></p>`

I temporarily got around this by running this function form the developer console ingame

```js
function fix_quest_log() {
    let folder = game.journal.directory.folders.find(f => f.name === "_fql_quests");

    folder.content.forEach( entry => {
        let data = entry.data;
        let content = data.content.replace(/((^\<p\>)|(\<\/p\>$))/mg, "");
        console.log(content)
        entry.update({ content: content}).then(e => {
            console.log("updated", e);
        })
    })
}

```

**Comment by sytone** [comment]
Thanks for the report. I’ll add a option to exclude transformation or import/export based on file name.

**Comment by sytone** [comment]
Fixed with commit cbd658a4037749196925553b7289585c3e96151a and released as version 0.4.0

**Comment by sytone** [comment]
Released as version 0.4.0, please update and thanks for providing feedback!
